packages/liveblocks-core/src/comments/comment-body.ts

```diff
@@ -632,3 +632,112 @@ export async function stringifyCommentBody(
 
   return blocks.join(separator);
 }
+
+export type TransformCommentBodyOptions<U extends BaseUserMeta = DU> = {
+  /**
+   * Which format to transform the comment body to.
+   */
+  format?: "html" | "json";
+  /**
+   * A function that returns info from user IDs.
+   */
+  resolveUsers?: (
+    args: ResolveUsersArgs
+  ) => OptionalPromise<(U["info"] | undefined)[] | undefined>;
+};
+
+export type CommentBodyJsonMention = {
+  type: "mention";
+  user: string;
+};
+export type CommentBodyJsonLink = CommentBodyLink;
+export type CommentBodyJsonText = CommentBodyText;
```

Comment:
I tried to avoid to mix types for different context. 
From my experience ( I may be wrong) I think it's a good thing to keep a clean separation of concern and use types legacy in this case. 

Even if a `CommentBodyJsonLink` is a `CommentBodyLink` it is not the same context. For us it's a shortcut for sure but we should pay attention to what we give to developers. 
`CommentBodyJsonLink` is the result after transforming the comment body. So I think we should be explicit enough to developers to let them know what is what. Even if the types are the same, `CommentBodyLink` should be used in the context  of the representation of what we get from the DB and `CommentBodyJsonLink` should be used in the context of the transformation of a comment body.

Does it makes sens to you or am I over-thinking here?